.github/workflows/main.yml

```diff
@@ -1,5 +1,9 @@
 name: Claude Code Review
 
+permissions:
```

Comment:
The patch improves security by specifying granular permissions instead of broad 'write-all'. It also uses a personal GitHub token instead of the default, which may provide more flexibility. However, ensure the PERSONAL_GITHUB_TOKEN has necessary permissions.

``` 
permissions:
  contents: read
  pull-requests: write
  issues: write
```